core/src/main/java/org/elasticsearch/search/rescore/QueryRescorerBuilder.java

```diff
@@ -78,6 +75,25 @@ public QueryRescorerBuilder(QueryBuilder<?> builder) {
     }
 
     /**
+     * Read from a stream.
+     */
+    public QueryRescorerBuilder(StreamInput in) throws IOException {
+        super(in);
```

Comment:
Like in this case I could replace the default constructor in RescoreBuilder with one that takes `(Integer windowSize)`.